_config.yml

```diff
@@ -166,6 +166,7 @@ devsearches:
   ## leading capital) for easier quick scanning
   "RPCs":
     - 'AbandonTransaction': "/en/developer-reference#abandontransaction"
+    - 'AbortRescan': "/en/developer-reference#abortrescan"
     - 'AddMultiSigAddress': "/en/developer-reference#addmultisigaddress"
     - 'AddNode': "/en/developer-reference#addnode"
     - 'AddWitnessAddress': "/en/developer-reference#addwitnessaddress"
```

Comment:
Oh, a segwit reference :thinking: It must have come over with the original fork and I didn't know what to do with it at the time. It shows as "Not Implemented" [here](https://dash-docs.github.io/en/developer-reference#addwitnessaddress), but I will remove it fully since there is no reason to keep it.